packages/react-native/package.json

```diff
@@ -37,12 +37,15 @@
   "dependencies": {
     "@honeybadger-io/core": "6.6.0",
     "@quiltt/core": "workspace:*",
-    "@quiltt/react": "workspace:*"
+    "@quiltt/react": "workspace:*",
+    "base-64": "1.0.0",
+    "react-native-url-polyfill": "2.0.0",
+    "react-native-webview": "13.10.2"
```

Comment:
Can we loosen the versions a bit so we don't have to have everyone update in case of a downstream security vulnerability? 

And assuming we need all these deps here?

Reply:
These are `dependencies` used by the package, we have looser requirements in the `peerDependencies` field.

But regardless,
- `base-64@v1.0.0` was released 4 years ago
- `react-native-url-polyfill@v2.0.0` was released 2 years

Besides `react-native-webview`, I'm fairly certain these packages won't cause any issues